Doc/library/uuid.rst

```diff
@@ -187,41 +187,38 @@ The :mod:`uuid` module defines the following functions:
 .. index:: single: getnode
```

Comment:
There is also this one to move. Can you check all `.. index::` usages in the file please?

Reply:
Sure, I'll do, thanks @picnixz